dask_geopandas/hilbert_distance.py

```diff
@@ -0,0 +1,261 @@
+import geopandas
+import numpy as np
+
+# from numba import jit # optional - do we want numba as a dependency for dask-geopandas?
+# ngjit = jit(nopython=True, nogil=True)
+
+
+def _hilbert_distance(gdf, total_bounds, p):
+
+    """
+    Calculate the hilbert distance for a GeoDataFrame based on the mid-point of
+    the bounds for each geom and total bounds of the collection of geoms
+
+    Based on: https://github.com/holoviz/spatialpandas/blob/
+    9252a7aba5f8bc7a435fffa2c31018af8d92942c/spatialpandas/dask.py#L172
+
+    Parameters
+    ----------
+    gdf          : GeoDataFrame
```

Comment:
```suggestion
    gdf : GeoDataFrame
```

Only a single space before (and after) the colon, see https://numpydoc.readthedocs.io/en/latest/format.html#parameters (that's the formatting issue Stefanie mentioned yesterday)

(and the same for the other docstrings)

Reply:
resolved in ce52794